Dynatrace/dynatrace-operator
# Adds run all e2e command/script + ENFORCE_ISTIO

# Description

Added a simple shell script that goes through our `make` targets for e2e testing and runs then 1-by-1. (using the `make` dependencies feature is not good because if 1 test fails then the others wont be run which is not what we want)

Also adds the `ENFORCE_ISITO` envvar which 
- if set to `true` then the tests that need istio will fail if istio is not present
- if not set then istio related steps are just skipped when istio is not present

## How can this be tested?
try `make ENFORCE_ISTIO="true" test/e2e/cloudnative/istio` should fail if istio is not in the cluster (try it also without the envvar)
try `make test/e2e` if it runs everything (it will take a loong time)

## Checklist
